### DISCUSSION TITLE
Filter by date range in the User reports

### DISCUSSION THREAD

**Discussion by CoumbaNDI** [Azure/api-management-developer-portal#2547] [Feature request]
Hello, 

When will the " filter by date range " feature be available in the new developer portal ?

**Comment by mikebudzynski** [comment]
Thanks for opening the feature request. We don't have an ETA for adding the filter by a custom date range feature.

**Comment by kevinpearce** [comment]
Hi - just wondering if this has perhaps been discussed/re-assessed in the last ~4years as still remains open?

eg. This is still a request from customers and the custom Widget route is far more complicated as would likely incur having to re-build a lot of the existing functionality just to be able to replicate the page with an additional filter.

Thanks
